plugins/remote-config/tests/index.js

```diff
@@ -1,4 +1,20 @@
-require('./add-remote-config.js');
-require('./fetch_remote_config.js');
-require('./condition-endpoints.js');
-require('./remote-config-endpoint.js');
\ No newline at end of file
+const plugins = require('../../pluginManager.js');
+
+try {
+    if (plugins.isPluginEnabled('ab-testing')) {
+        require('./ab_rc-method.js');
```

Comment:
Let's just move these tests to ab testing plugin to avoid using conditionals here

There're tests for fetch remote config in ab testing plugin https://github.com/Countly/countly-enterprise-plugins/blob/ab-testing-cycle9/plugins/ab-testing/tests/fetch_remote_config.js